src/azul/__init__.py

```diff
@@ -1772,45 +1772,66 @@ def docker_image_gists_path(self) -> Path:
 
     blocked_user_agents_custom_regex_term = 'blocked_user_agents_custom'
 
-    waf_rate_rule_name = 'RateRule'
+    waf_rate_rule_name = 'rate_limit'
 
-    waf_rate_alarm_rule_name = 'RateAlarmRule'
+    waf_rate_alarm_rule_name = 'rate_limit_alarm'
 
-    waf_rate_rule_period = 300  # seconds; this value is fixed by AWS
+    @frozen(kw_only=True)
+    class RateLimit:
+        #: Name of the WAF rule
+        name: str
 
-    waf_rate_rule_retry_after = 30  # seconds
+        #: Number of requests per evaluation period
+        value: int
 
-    waf_rate_rule_limit = 1000
+        #: Evaluation window in seconds. Note, the rate is checked every 10
+        #: seconds. This property specifies how far back from the current time
+        #: to consider when calculating the rate.
```

Comment:
```suggestion
        #: WAF rate limit evaluation window in seconds
```

The rest is evident from AWS documentation.